lib/images.sh

```diff
@@ -24,8 +24,8 @@ else
 fi
 
 # Image url and checksum
-export IMAGE_URL=http://$PROVISIONING_URL_HOST/images/${IMAGE_NAME}
-export IMAGE_CHECKSUM=http://$PROVISIONING_URL_HOST/images/${IMAGE_NAME}.md5sum
+export IMAGE_URL=http://$BARE_METAL_PROVISIONER_URL_HOST/images/${IMAGE_NAME}
```

Comment:
Curlys missing, and quote the right hand side.

Reply:
done